firefox-ios/Client/Frontend/Autofill/Address/Edit/EditAddressViewController.swift

```diff
@@ -203,6 +206,20 @@ class EditAddressViewController: UIViewController, WKNavigationDelegate, WKScrip
     }
 }
 
+// MARK: - KeyboardHelperDelegate
+extension EditAddressViewController: KeyboardHelperDelegate {
```

Comment:
@issammani lets move this out of extension and into the ViewController considering we don't use them in this fashion.

Reply:
Done ! Updated keyboard handling code to be inside the viewcontroller in  https://github.com/mozilla-mobile/firefox-ios/pull/22498/commits/77716ea28247c5f20a1d04468f9433d72104cbfb. Also fixed an issue with the layout being weird when the remove button is hidden in  https://github.com/mozilla-mobile/firefox-ios/pull/22498/commits/13953940ed6ad217a4313779b38455ee114a5828